bigquery/table.go

```diff
@@ -147,6 +147,99 @@ type TableMetadata struct {
 	//   - '': empty string. Default to case-sensitive behavior.
 	// More information: https://cloud.google.com/bigquery/docs/reference/standard-sql/collation-concepts
 	DefaultCollation string
+
+	// PrimaryKey constraint on a table's columns.
+	// Present only if the table has a primary key.
+	// The primary key is not enforced.
+	PrimaryKey *PrimaryKey
+
+	// ForeignKeys represent a list of foreign keys constraints.
+	// The foreign key is not enforced.
+	ForeignKeys []*ForeignKey
+}
+
+// PrimaryKey represents the primary key constraint on a table's columns.
+type PrimaryKey struct {
+	// Columns that compose the primary key constraint.
+	Columns []string
```

Comment:
Question:  are there ordering constraints here (and in FK) we need to document?  Or does the order of columns not matter?

Reply:
AFAIK the ordering doesn't matter, but I'll double check internally